README.md

```diff
@@ -273,6 +290,32 @@ associative array in the first argument:
 </div>
 ```
 
+You can also pass parameters to actions:
```

Comment:
How about we link `parameters` to the page on the Stimulus docs about action parameters.